sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  intel/llvm#14520 intel/llvm#14485 intel/llvm#14510 intel/llvm#14483
+  intel/llvm#14487 intel/llvm#14488
+- Added support for `sycl::vec::convert` to/from `vec<bfloat16, N>`.
+  intel/llvm#14105 intel/llvm#14085
+- Deprecated `marray<bool, n>::operator++/--`. intel/llvm#13443
+- Deprecated `accessor::get_multi_ptr` for non-device accessors. intel/llvm#13443
+- Moved ESIMD named barrier APIs out of `experimental` namespace. intel/llvm#13704
+- Implemented latest revision of [`sycl_ext_oneapi_free_function_queries`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/supported/sycl_ext_oneapi_free_function_queries.asciidoc)
+  extension. intel/llvm#13257
+- Extended `sycl-ls --verbose` to print device's UUID, information about its
+  sub- and sub-sub- devices and its architecture. intel/llvm#13999 intel/llvm#13976
+- Added support for compile-time properties to `copy_to` and `copy_from` ESIMD
+  APIs. intel/llvm#13586
+- Switched `experimental::printf` implementation to use non-variadic interface
+  by default. This should improve usability when printing `float` values on
+  devices that doesn't support `fp64` aspect by disabling `float` -> `double`
+  promotion in `printf` arguments. intel/llvm#13055
+- Added a diagnostic if `slm_init` ESIMD API is called more than once in
+  a kernel. intel/llvm#12804
+- Updated implementation of
+  [`sycl_ext_oneapi_device_architecture`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc)
+  extension to return `unknown` enumerator on an unsupported HW. intel/llvm#14190
+- Extended list of known Intel GPU architectures available through
+  [`sycl_ext_oneapi_device_architecture`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc)
+  extension. intel/llvm#13520 intel/llvm#14582
+- Extended mechanism to clear in-memory cache in heavy tests to also work on
+  `opencl` backend. intel/llvm#14119
+- Moved bit shift and rotate ESIMD functions out of `experimental` namespace.
+  intel/llvm#13545
+- moved `rdtsc` ESIMD function out of `experimental` namespace. intel/llvm#13417
+- Added check for template argument `N` of `media_block_load` ESIMD API. intel/llvm#13668
+- Enhanced deprecation message for `sub_group::barrier` to indicate which API
+  should be used instead. intel/llvm#13276
+- Added deprecation messages for `image_max_array_size` and `opencl_c_version`
+  device info queries. intel/llvm#13279
+- Updated [`sycl_ext_intel_device_info`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/supported/sycl_ext_intel_device_info.md)
+  extension implementation to throw synchronious exception with
+  `feature_not_supported` error code. intel/llvm#14788
+- Reduced startup overhead on `libsycl.so` loading by outlining SYCL JIT
```

Comment:
```suggestion
- Reduced startup overhead of `libsycl.so` by outlining SYCL JIT
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706